float32 velocity
float32 damping_distance
Waypoint[] mission_waypoint
